cristin-import/src/test/resources/features/DegreePhdRules.feature

```diff
@@ -0,0 +1,9 @@
+Feature:
+  Mapping rules for Degree_phds
+
+  Background:
+    Given a valid Cristin Result with secondary category "DRGRADAVH"
+
+  Scenario: Cristin Result with type "Drgradavh" is converted to an NVA entry with PublicationInstance of type "DegreePhd".
```

Comment:
Use the file in Confluence to file the english name so that it is readable for Terje. 
E.g. Cristin Result  "Ph.D thesis" is converted to an NVA entry with type "DegreePhd"